app/controllers/posts_controller.rb

```diff
@@ -10,10 +10,13 @@ def new
 
```

Comment:
Please ensure that you solve N+1 problem when fetching all `posts` and their `comments` for a `user` according to the project requirement. Example in the posts#index actions you should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`

Reply:
Thank you for the feedback @micheaol . I'm fixing that right away.

Reply:
About the failure of the request specs, you will need to populate a post and user in your test db locally.

Reply:
@AdornChoga ,

That was done, but some tests still failed. I suggest that you double-check.

Reply:
@micheaol okay thanks. Let me double-check.